docs/developer-docs/latest/setup-deployment-guides/configurations.md

```diff
@@ -1,1775 +1,70 @@
-
-```js
-handler: () => new Date().getHours() === 17;
-```
-
-The `handler` function receives the authenticated user, so it can verify conditions on the user:
-
-```js
-const condition = {
-  displayName: 'Email address from strapi.io',
-  name: 'email-strapi-dot-io',
-  async handler(user) {
-    return user.email.includes('@strapi.io');
-  },
-};
-```
-
-For more granular control, the `handler` function can also return a query object:
-
-```js
-const condition = {
-  displayName: 'price greater than 50',
-  name: 'price-gt-50',
-  async handler(user) {
-    return { price: { $gt: 50 } };
-  },
-};
-```
-
-#### Registering conditions
-
-To be available in the admin panel, conditions should be declared and registered in the [`./config/functions/bootstrap.js`](/developer-docs/latest/setup-deployment-guides/configurations.md#bootstrap) file. Register a single condition with the `conditionProvider.register()` method:
-
-```js
-module.exports = async () => {
-  await strapi.admin.services.permission.conditionProvider.register({
-    displayName: 'Billing amount under 10K',
-    name: 'billing-amount-under-10k',
-    plugin: 'admin',
-    handler: { amount: { $lt: 10000 } },
-  });
-};
-```
-
-To register multiple conditions, defined as an array of [condition objects](#declaring-new-conditions), use `conditionProvider.registerMany()`:
-
-```js
-const conditions = [
-  {
-    displayName: "Entity has same name as user",
-    name: "same-name-as-user",
-    plugin: "name of a plugin if created in a plugin"
-    handler: (user) => {
-      return { name: user.name };
-    },
-  },
-  {
-    displayName: "Email address from strapi.io",
-    name: "email-strapi-dot-io",
-    async handler(user) {
-      return user.email.includes('@strapi.io');
-    },
-  }
-];
-
-module.exports = async () => {
-  // do your boostrap
-
-  await strapi.admin.services.permission.conditionProvider.registerMany(conditions);
-};
-```
+- [middlewares](/developer-docs/latest/setup-deployment-guides/configurations/optional/middlewares.md)
+- [functions](/developer-docs/latest/setup-deployment-guides/configurations/optional/functions.md)
+- [public assets](/developer-docs/latest/setup-deployment-guides/configurations/optional/public-assets.md)
+- [API calls](/developer-docs/latest/setup-deployment-guides/configurations/optional/api.md)
+- the [environment and its variables](/developer-docs/latest/setup-deployment-guides/configurations/optional/environment.md)
+- [Single Sign-On](/developer-docs/latest/setup-deployment-guides/configurations/optional/sso.md) <GoldBadge link="https://strapi.io/pricing-self-hosted/" withLinkIcon />
+- [Role-Based Access Control](/developer-docs/latest/setup-deployment-guides/configurations/optional/rbac.md) <BronzeBadge link="https://strapi.io/pricing-self-hosted"/> <SilverBadge link="https://strapi.io/pricing-self-hosted"/> <GoldBadge link="https://strapi.io/pricing-self-hosted" withLinkIcon/>
```

Comment:
Items of list list are not in the same order as in the main TOC, it would make more sense to follow the same logic in both, don't you think @pwizla ?

Reply:
Definitely! Good catch :-)